doc/rules/member_ordering.md

```diff
@@ -8,7 +8,7 @@ member-ordering
 
 Enforces member ordering.
 
-The value for order may be an array consisting of the following strings:
+The value for `order` may be an array consisting of the following strings:
```

Comment:
I guess it should be mentioned that following is the default order. Or if it isn't - add a list with default ordering.

Reply:
I have added a description, that its a default order